src/overlays/actors/ovl_En_Skb/z_en_skb.c

```diff
@@ -409,14 +409,14 @@ void func_80AFD7B4(EnSkb* this, GlobalContext* globalCtx) {
     }
     this->unk_280 = 1;
     this->actor.flags &= ~1;
-    func_80032E24(&this->unk_28C, 18, globalCtx);
+    BodyBreak_Alloc(&this->bodyBreak, 18, globalCtx);
     this->unk_283 |= 4;
     EffectSsDeadSound_SpawnStationary(globalCtx, &this->actor.projectedPos, NA_SE_EN_STALKID_DEAD, 1, 1, 0x28);
     EnSkb_SetupAction(this, func_80AFD880);
 }
 
 void func_80AFD880(EnSkb* this, GlobalContext* globalCtx) {
-    if (func_8003305C(&this->actor, &this->unk_28C, globalCtx, 1) != 0) {
+    if (BodyBreak_SpawnParts(&this->actor, &this->bodyBreak, globalCtx, 1)) {
         if (this->actor.scale.x == 0.01f) {
             Item_DropCollectibleRandom(globalCtx, &this->actor, &this->actor.world.pos, 0x10);
         } else {
```

Comment:
Github doesn't let me take the range of lines I want to make a suggestion, so I'm adding a screenshot.
Could you merge the `else` and the `if`? Does it match?
![image](https://user-images.githubusercontent.com/7416381/116164465-b593d800-a6c7-11eb-8e37-7da162e2f7da.png)

Reply:
prob can but its a bit out of scope from what im doing here, this kind of thing will get fixed in other passes of this actor.
Will try it anyway tho